research/src/pages/contribute.mdx

```diff
@@ -81,7 +81,7 @@ Add the `$schema` key pointing to our `design-system.schema.json5` schema and co
 {
   $schema: '../schemas/design-system.schema.json5',
   name: 'Ant Design',
-  description: 'A design system with values of Nature and Determinacy for better user experience of enterprise applications.',
+  description: 'An enterprise-class UI design language and React UI library.',
```

Comment:
I've updated this description with the tagline from the Ant Design Readme. I liked it cuz it's a little shorter and reduces the screen size at which as scrollbar is displayed for this code sample